script/upgrades/MU03/MU03Checks.sol

```diff
@@ -17,6 +17,7 @@ import { IBroker } from "mento-core-2.2.0/interfaces/IBroker.sol";
 import { IERC20Metadata } from "mento-core-2.2.0/common/interfaces/IERC20Metadata.sol";
 
 import { BiPoolManagerProxy } from "mento-core-2.2.0/proxies/BiPoolManagerProxy.sol";
+import { SortedOraclesProxy } from "mento-core-2.0.0/proxies/SortedOraclesProxy.sol";
```

Comment:
Let's use the `2.2.0` consistently in `MU03`, even tho in practice this ends up being the same contract in this case, if we mix up different versions we might introduce subtle bugs.

Reply:
good catch! it was the co-pilot 😤